packages/presentation/lib/bloc/home_bloc.dart

```diff
@@ -30,7 +34,7 @@ class _HomeBloc extends BlocImpl implements HomeBloc {
     updateData();
 
     _screenData.isPalindrome =
-        await _palindromeUseCase.isPalindrome(_screenData.stringInput);
+        await _palindromeUseCase.call(_screenData.stringInput);
```

Comment:
fyi: it's not obvious to call `call` method explicitly. u can call _palindromeUseCase(_screenData.stringInput)